Sources/ApolloCodegenLib/IR/ScopeDescriptor.swift

```diff
@@ -3,9 +3,6 @@ import OrderedCollections
 
 extension IR {
 
-  // TODO: Write tests that two inline fragments with same type and inclusion conditions,
```

Comment:
Done in https://github.com/apollographql/apollo-ios/pull/3176/files#diff-75a0a416f524d6139314ef8b30651767a446ba7f6a1dbc0e00a4c64f130251d1R2232.

Reply:
I'm thinking we should also be adding tests for this behavior to the IR building tests, not just the `SelectionSetTemplateTests`. Making sure the actual IR is structured properly is important and having those tests makes it a LOT easier to debug and figure out the cause of bugs than just the template tests.